2-mandatory/6-writing-tests-advanced.js

```diff
@@ -55,6 +57,14 @@ function formatCourseworkResult(trainee) {
     score: 63
   }
 */
```

Comment:
Great job on these tests!
My only comment would be that most of your tests have the same name: "returns correct coursework result string for trainee".
It's usually a good idea to have a different name for each test - which describes what is being tested. Then, if one of the tests fails - the developer can easily see which test the error message is referring to 😄